proposal-glacier/README.md

```diff
@@ -0,0 +1,68 @@
+# Summary
```

Comment:
Could we add some instructions on how to run the examples?

Reply:
Done (: